### PR TITLE
Header & single updates

### PR DESCRIPTION
**Pull request by ashleyshaw**
#6

### CODE REVIEW

**Comment by carolinan** [comment]
Thank you for working on this.

I can't remember exactly why we chose the columns block for the header. Now, on desktop width, when the post has no featured image, the space is empty, and it looks strange because the first column's width is still the same.

the post title in the large header is right aligned, left of a large open space.




There are two submissions for the content part with the sidebar:
https://github.com/WordPress/community-themes/issues/6#issuecomment-1582569717

**Comment by carolinan** [comment]
Out of the two suggestions, I think the version where the tags, navigation links, and comments are _next_ to the sidebar, and not _below_ it, works better.
We have to make sure that the sidebar uses headings, not paragraphs, as section headers above the blocks.

File: blue-note/templates/single.html [hunk 1]
```diff
@@ -1,11 +1,125 @@
...
+<!-- /wp:group --></div>
+<!-- /wp:comments --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"33.33%","style":{"spacing":{"blockGap":"var:preset|spacing|60"}}} -->
+<div class="wp-block-column" style="flex-basis:33.33%"><!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase","fontSize":"22px"}}} -->
+<p style="font-size:22px;text-transform:uppercase"><strong>Latest Post</strong></p>
+<!-- /wp:paragraph -->
+
```
Comment (carolinan) [inline blue-note/templates/single.html]:
```suggestion
<div class="wp-block-group"><!-- wp:heading {"style":{"typography":{"fontSize":"22px","textTransform":"uppercase","fontStyle":"normal","fontWeight":"700"}}} -->
<h2 class="wp-block-heading" style="font-size:22px;font-style:normal;font-weight:700;text-transform:uppercase">Latest Post</h2>
<!-- /wp:heading -->
```

**Comment by MaggieCabrera** [comment]
> I can't remember exactly why we chose the columns block for the header. 

I believe we had issues when the text is longer than the featured image, and that wouldn't stretch to fit the content.

**Comment by carolinan** [comment]
We should probably remove the newsletter button, unless we want to add support for a third party news letter plugin.

File: blue-note/templates/single.html [hunk 2]
```diff
@@ -1,11 +1,124 @@
...
+<div class="wp-block-buttons"><!-- wp:button {"style":{"typography":{"fontSize":"22px"}}} -->
+<div class="wp-block-button has-custom-font-size" style="font-size:22px"><a class="wp-block-button__link wp-element-button">Subscribe</a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}}},"textColor":"contrast","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-contrast-color has-text-color has-link-color"><!-- wp:paragraph {"style":{"typography":{"fontSize":"22px","textTransform":"uppercase"}}} -->
+<p style="font-size:22px;text-transform:uppercase"><strong>Categories</strong></p>
+<!-- /wp:paragraph -->
```
Comment (carolinan) [inline blue-note/templates/single.html]:
```suggestion
<div class="wp-block-group has-contrast-color has-text-color has-link-color"><!-- wp:heading {"style":{"typography":{"fontSize":"22px","textTransform":"uppercase","fontStyle":"normal","fontWeight":"700"}}} -->
<h2 class="wp-block-heading" style="font-size:22px;font-style:normal;font-weight:700;text-transform:uppercase">Categories</h2>
<!-- /wp:heading -->
```

**Comment by carolinan** [comment]
We can also increase the spacing between the latest posts in the sidebar by increasing the line-height. That works as long as we are only showing the post titles...

**Comment by MaggieCabrera** [comment]
I made a few changes and I think we can bring this one in and iterate

**Review by MaggieCabrera — APPROVED** [review]

**Pull request merged**